### PR TITLE
Create it_it.json

### PR DESCRIPTION
**Pull request by Roby1164**

### CODE REVIEW

File: shared-sources/src/main/resources/assets/inventoryprofilesnext/lang/it_it.json [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+    "inventoryprofiles.config.description.auto_refill_match_nbt_type":                   "Questo controlla come viene confrontato NBT.\n'Esatto' significa che NBT deve corrispondere completamente.\n'Consenti extra' significa che verranno confrontati solo i tag uguali.",
+    "inventoryprofiles.config.description.auto_refill_ignore_nbt_for_buckets":           "Se questo è impostato su falso, i secchi di pesci e Axolotl si riempiranno solo se contengono lo stesso tipo di pesci/Axolotl.",
+    "inventoryprofiles.config.description.auto_refill_match_any_food":                   "Quando questo è impostato su vero quando una pila di cibo è esaurita, qualsiasi altro cibo può essere usato per riempirlo.\n\nNOTA:\nQuesto ordinerà gli alimenti per saturazione, cioè e la mela d'oro incantata sarà la prima scelta. E così via...",
+    "inventoryprofiles.config.description.auto_refill_match_harmful_food":               "Quando questo e quanto sopra sono impostati su true, anche gli alimenti dannosi, come il "pesce palla", verranno utilizzati per riempire le pile di cibo esaurite.",
```
Comment (blackd) [inline shared-sources/src/main/resources/assets/inventoryprofilesnext/lang/it_it.json]: " in the strings has to be escaped with \ before them

**Comment by blackd** [comment]
Thank you very much!

there are some formatting problems you can see all of them here https://github.com/blackd/Inventory-Profiles/blob/d59f779623d8a5ee07404dc88e33a839126142bd/shared-sources/src/main/resources/assets/inventoryprofilesnext/lang/it_it.json

I can fix them but since I don't speak Italian I would prefer if you do it.

**Review by Roby1164 — COMMENTED** [review]
Ok, fixed

**Review by Roby1164 — COMMENTED** [review]
Saw another error fixed

**Pull request merged**